### PR TITLE
Loading request airtable tracking

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

**Comment by hanars** [comment]
Note: hold off on merging until move to the correct base

File: seqr/views/apis/anvil_workspace_api_tests.py [hunk 1]
```diff
@@ -432,12 +454,22 @@ def test_create_project_from_workspace(self, mock_tempfile, mock_mv_file, mock_f
...
         self.assertEqual(responses.calls[0].request.url, '{}/api/v1/dags/seqr_vcf_to_es_AnVIL_WES_v0.0.1/dagRuns'.format(MOCK_AIRFLOW_URL))
         self.assertEqual(responses.calls[0].request.method, "GET")
         self.assertEqual(responses.calls[0].request.headers['Authorization'], 'Bearer {}'.format(MOCK_TOKEN))
         self.assertEqual(responses.calls[0].response.json(), DAG_RUNS_RUNNING)
 
+        # Airtable record created with correct status
+        self.assertDictEqual(json.loads(responses.calls[1].request.body), {'records': [{'fields': {
+            'Requester Name': 'Test Manager User',
+            'Requester Email': 'test_user_manager@test.com',
+            'AnVIL Project URL': f'http://testserver/project/{project2.guid}/project_page',
```
Comment (ShifaSZ) [inline seqr/views/apis/anvil_workspace_api_tests.py:467]: As above.

**Review by ShifaSZ — CHANGES_REQUESTED** [review]

File: seqr/views/apis/anvil_workspace_api_tests.py [hunk 2]
```diff
@@ -365,6 +374,17 @@ def test_create_project_from_workspace(self, mock_tempfile, mock_mv_file, mock_f
         self.assertDictEqual(json.loads(responses.calls[4].request.body), {})
         self.assertEqual(responses.calls[4].request.headers['Authorization'], 'Bearer {}'.format(MOCK_TOKEN))
 
+        # create airtable record
+        self.assertDictEqual(json.loads(responses.calls[5].request.body), {'records': [{'fields': {
+            'Requester Name': 'Test Manager User',
+            'Requester Email': 'test_user_manager@test.com',
+            'AnVIL Project URL': f'http://testserver/project/{project.guid}/project_page',
```
[the remark this answers is not in the document]
Comment (hanars) [reply seqr/views/apis/anvil_workspace_api_tests.py:381]: this is consistant with the existing usage in the test (`project.guid` is used a few times in this code here), so its out of scope to change this

File: seqr/views/apis/anvil_workspace_api_tests.py (hunk 1 above)
Comment (hanars) [reply seqr/views/apis/anvil_workspace_api_tests.py:467]: same reply as above

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**